tests/framework/installer/install_helper.go

```diff
@@ -475,3 +478,134 @@ func IsAdditionalDeviceAvailableOnCluster() bool {
 	logger.Info("No additional disks found on cluster")
 	return false
 }
+
+// ************************************************************************************************
+// CockroachDB functions
+// ************************************************************************************************
+func (h *InstallHelper) InstallCockroachDB(systemNamespace, namespace string, count int) error {
```

Comment:
I disagree with having all of these integration test functions in the same file. Over time, there will be more backends and we can't have them including their functions like this. We should consider splitting them out into their own namespaces.

Reply:
I agree that this isn't the long term solution for integration tests with multiple providers.  We'd like to even test them in parallel also, so splitting them into multiple pipelines or something would be ideal.  Not sure how easy that is though, that would require some education/ramp-up.  Are you against including this integration test in this PR without full refactoring/isolation for multiple providers integration tests?

Reply:
> Are you against including this integration test in this PR without full refactoring/isolation for multiple providers integration tests?

Yes. I think if we're going to redo all of this e2e code in the next few weeks, we should just do it the "right" way the first time. With these commits, we should be establishing a pattern for all other initial back-end implementations beyond cockroach/minio to follow. My concern is that this test goes in and the urgency of the integration test refactor goes away, reducing the probability of actually doing it, and creating a larger amount of work in the future.